### PR TITLE
Added C++ code in usaco-1059.mdx

### PR DESCRIPTION
**Pull request by mshahanwaz**
Added C++ code for Do You Know Your ABCs? -bronze problem in usaco-1059.mdx

_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

File: solutions/usaco-1059.mdx [hunk 1]
```diff
@@ -8,6 +8,27 @@ author: Sathvik Chundru
...
 <LanguageSection>
+	
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	int nums[7];
```
Comment (SansPapyrus683) [inline solutions/usaco-1059.mdx:19]: change to vector

File: solutions/usaco-1059.mdx [hunk 2]
```diff
@@ -8,6 +8,27 @@ author: Sathvik Chundru
...
+	for(int i = 0; i < 7; i++) {
+		cin >> nums[i];
+	}
+	sort(nums, nums + 7);
+	int a = nums[0], b = nums[1];
```
Comment (SansPapyrus683) [inline solutions/usaco-1059.mdx:24]: separate statements on one line

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-1059.mdx (hunk 1 above)
Comment (Aadit-Ambadkar) [reply solutions/usaco-1059.mdx:19]: why?

File: solutions/usaco-1059.mdx (hunk 2 above)
Comment (Aadit-Ambadkar) [reply solutions/usaco-1059.mdx:24]:
```suggestion
	int a = nums[0];
	int b = nums[1];
```

File: solutions/usaco-1059.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-1059.mdx:19]: harder to peek inside

Comment (bqi343) [reply solutions/usaco-1059.mdx:19]: not sure what you mean by that, it's fine as is

Comment (SansPapyrus683) [reply solutions/usaco-1059.mdx:19]: it's just harder to print out arrays

**Review by ryanchou-dev — APPROVED** [review]

**Review by TruaShamu — APPROVED** [review]
looks good to me

**Pull request merged**